hub/policy/src/main/java/uk/gov/ida/hub/policy/contracts/AttributeQueryContainerDto.java

```diff
@@ -13,6 +15,7 @@
     private String id;
     private String issuer;
     private boolean onboarding;
+    private Optional<CountrySignedResponseContainer> countrySignedResponse;
```

Comment:
I think this new field is unnecessary. This dto is [constructed from the Response from saml-engine](https://github.com/alphagov/verify-hub/pull/381/files#diff-35fb4a887d81aafec291a5afe9a0e946R95), but the dto that saml-engine [uses doesn't have this field](https://github.com/alphagov/verify-hub/blob/master/hub/saml-engine/src/main/java/uk/gov/ida/hub/samlengine/contracts/AttributeQueryContainerDto.java#L26). 

The country-response is wrapped up in the attribute-query inside the `samlRequest` field of this dto.